src/assets/ba_data/python/babase/_plugin.py

```diff
@@ -171,6 +171,10 @@ def on_app_running(self) -> None:
 
                 _error.print_exception('Error in plugin on_app_running()')
 
+        # This seems illegal
+        assert _babase.app.plus is not None
+        _babase.app.plus.rpc_thread.start()
```

Comment:
why starting thread here i this file?

Reply:
Couldn't find a better place really, and starting next to plugins seemed perfect (as immediate as I want it to be, while everything we need is already prepared/initiated). Will test different places later to see what makes the most sense.